cogs/induct.py

```diff
@@ -73,11 +74,9 @@ async def on_member_update(self, before: discord.Member, after: discord.Member)
 
         async for message in after.history():
             message_is_introduction_reminder: bool = (
```

Comment:
Missing explicit cast to bool. (A `# noqa: COM812` comment will need to be added after this change)

Fix:
```python
message_is_introduction_reminder: bool = bool(
    ("joined the " in message.content)
    and (...
)
```